text/0192-remove-constructs-compat.md

```diff
@@ -1096,3 +993,95 @@ for constructs 10.x.
   <https://github.com/aws/aws-cdk/issues/8909>
 - [ ] Updates to Developer Guide (add 2.x section)
 - [ ] Updates to READMEs across the library (add 2.x section)
+
+## Rejected Work
+
+The following section includes sections that were rejected from inclusion in this project.
+
+### 10-CONSTRUCT-NODE
+
+Originally, we wanted to rename the `node` property to `construct` (`bucket.construct` instead of `bucket.node`).
+The motivation is described below in the original text.
+
+We eventually decided not to include this change because languages like .NET do not allow properties
+to have the same name as the class `Construct` and `Construct` (this name is preserved in .NET to the constructor),
+and jsii does not have any support for per-language naming.
+
+References:
+
+- Pull request to change all occurances of `node` to `construct` in the AWS CDK codebase: <https://github.com/aws/aws-cdk/pull/9557>
+- Commit which reverts this change after jsii packaging failure: <https://github.com/aws/aws-cdk/commit/b6c1c8098c54160a09cae778d521fb1d3e6416f3>
+
+The original text follows:
```

Comment:
Might read better if you quote the following section using '> '

Reply:
Fixed